web/styles/dark_theme.css

```diff
@@ -796,7 +796,7 @@
     #subscription_overlay #stream-creation .settings-sticky-footer,
     #groups_overlay #user-group-creation .settings-sticky-footer {
         border-color: hsl(0deg 0% 0% / 20%);
-        background-color: hsl(0deg 0% 0% / 20%);
+        background-color: hsl(212deg 28% 15%);
```

Comment:
This changes color for a a lot of elements, can we just keep it scoped to the tables we want to fix the background for. Also, looks like there is white border in you video when drag the table row, would be great to remove that too.

Reply:
ok ,I will do the required changes wherever needed and will inform you , once its done!!
Thank You : )

Reply:
@amanagr 
color changes scoped to tables we want to fix the background color for
But i was not able to solve the problem of that white border while dragging the table row 
I was not able to find the CSS for which this problem keeps occuring 
But as per the issue #26480  I have solved it 
It would be great , if you review this once again
Thank You : )

Reply:
Well, I cannot approve this PR until my comment is addressed.

Reply:
@amanagr 
I would fix that white border issue 
but could you tell me whether the other changes I  made are correct  and have no problem ? 
Thank You: )

Reply:
> This changes color for a lot of elements, can we just keep it scoped to the tables we want to fix the background for.

Reply:
@amanagr
color changes scoped to tables we want to fix the background color for
But i was not able to solve the problem of that white border while dragging the table row
I was not able to find the CSS for which this problem keeps occuring
But as per the issue https://github.com/zulip/zulip/issues/26480 I have solved it
It would be great , if you review this once again
Thank You : )